docs/docs/tutorial/cli-client.md

```diff
@@ -61,7 +61,7 @@ Once configured, you can call the API operations using high-level commands gener
 
 {{ asciinema("../../terminal/restish-call.cast", rows="20") }}
 
-See the help commands like `restish tutorial --help` or `restish tutorial get-greeting --help` for more details. If you set up command-line completion, you can also use tab to see all available commands.
+See the help commands like `restish tutorial --help` or `restish tutorial get-greeting-by-name --help` for more details. If you set up command-line completion, you can also use tab to see all available commands.
```

Comment:
@alexanderilyin hmm, if you look at https://huma.rocks/tutorial/your-first-api/#__codelineno-8-29 and follow the tutorial it will indeed be called `get-greeting`, but if you skip that step you can wind up with the wrong command name that is auto-generated. 🤔